lib/omnibus/fetchers/git_fetcher.rb

```diff
@@ -203,7 +203,7 @@ def contains_revision?(rev)
     #   the shellout object
     #
     def git(command)
-      shellout!("git #{command}", cwd: project_dir)
+      shellout!("git -c core.autocrlf=false #{command}", cwd: project_dir)
```

Comment:
@jkeiser I'm fine with that, just wanted to bring it up for discussion :) (I'm +1 on a comment as well)